apps/Gruntfile.js

```diff
@@ -32,8 +32,7 @@ var envOptions = {
   minify: (process.env.MOOC_MINIFY === '1'),
   localize: (process.env.MOOC_LOCALIZE === '1'),
   dev: (process.env.MOOC_DEV === '1'),
-  digest: (process.env.MOOC_DIGEST === '1')
-
+  digest: false // Handled by Rails now
```

Comment:
I think we can go ahead and rip out all the Gruntfile digest code.  It should mostly be an inverse of this change to `Gruntfile.js`: https://github.com/code-dot-org/code-dot-org/pull/2992/files